lib/filetests/src/lib.rs

```diff
@@ -76,7 +76,30 @@ pub fn run(verbose: bool, files: &[String]) -> TestResult {
     }
 
     runner.start_threads();
-    runner.run()
+    runner.run(None, None)
+}
+
+/// Used for 'pass' subcommand.
+/// Commands are interpreted as test and executed.
+///
+/// Directories are scanned recursively for test cases ending in `.clif`.
+///
+pub fn run_pass(
```

Comment:
I wonder if `run_passes` would be a better name, since it can actually run a vector of them.